### DISCUSSION TITLE
Support templates in automation editor for choose actions

### DISCUSSION THREAD

**Discussion by hagenuck1** [home-assistant/frontend#9584] [Automations / scripts / scenes] [answered]
Hello,
unfortunately you get the "templates not supported" message if you try to insert a template in any field under a choose action in the automation editor. Now that you can set trigger names and can combine very large automations this does not only affects a single action (notify with relative time as a template for example), but the whole action which are multiple conditions in a choose. 

Would be great if only the action which contains the template would have to been edited in yaml and not the whole "choose" action.

Template editing was deactivated for good reasons in this PR: #8581

But now with choose actions and those "condition: trigger" it's not that great experience for that.
Greets :)

**Comment by pascallj** [comment, marked as answer]
I don't see any crosslinking between PR's and discussions, so I'll add it here manually.

This is now added in 20210825.0 via #9883
